pkg/runs/runsDownload.go

```diff
@@ -133,11 +148,25 @@ func nameDownloadFolder(run galasaapi.Run, runName string, timeService utils.Tim
 
 func downloadArtifactsToDirectory(apiServerUrl string,
 	directoryName string,
-	reRun galasaapi.Run,
+	run galasaapi.Run,
 	fileSystem utils.FileSystem,
 	forceDownload bool,
-	console utils.Console) error {
-	runId := reRun.GetRunId()
+	console utils.Console,
+	runDownloadTargetFolder string,
+) error {
+
+	runId := run.GetRunId()
+
+	// We want to base the directory we download to on the destination folder.
+	// If the destination folder is "." (current folder/relative)
+	// then ignore, as prefixing with a "." just adds noise when we log and
+	// print out the path.
+	if runDownloadTargetFolder != "." {
+		directoryName = runDownloadTargetFolder + "/" + directoryName
```

Comment:
Might be worth using `filepath.Join` to join path elements with an OS-specific separator?

Reply:
Yep. Used that. Good suggestion. Thanks.